leaf-ui/js/object/InputAnalysis.js

```diff
@@ -21,12 +21,16 @@ function getAnalysisValues(analysisInterface){
 	analysisInterface.elementList = getElementList();
 	
 	//Data required for 2. Explore Possible Next States
+	//Erase old initialAssignedEpoch data
```

Comment:
This comments are no accurate anymore?

Reply:
True... don't know why I left there... will erase.